migrations/src/migrations.rs

```diff
@@ -6,6 +6,7 @@ mod migration_0001_accounts;
 mod migration_0002_pilots;
 mod migration_0003_permissions;
 
+#[must_use]
```

Comment:
So I skipped one of the other ones that annotated must_use, but why are we adding these to a few functions? I can't figure out what's special about the functions that were annotated.

Reply:
I'm just following Clippy here. If clippy missed one, thats just a false-negative.
I didn't check if we missed one, but `#[must_use]` is supposed to be placed on all return values that aren't `#[must_use]` by default, like `Result` and similar things.

Reply:
Found the lint... still processing what I think of it.